operating/manage-user-access/enable-pki-authentication.adoc

```diff
@@ -0,0 +1,23 @@
+[id="enable-pki-authentication"]
+= Enabling PKI authentication
+include::modules/common-attributes.adoc[]
+:context: enable-pki-authentication
+
+toc::[]
+
+[role="_abstract"]
+If you use an enterprise certificate authority (CA) for authentication, you can configure {product-title} (RHACS) to authenticate users by using their personal certificates.
+
+After you configure PKI authentication, users and API clients can log in using their personal certificates.
+Users without certificates can still use other authentication options, including API tokens, the local administrator password, or other authentication providers.
+PKI authentication is available on the same port number as the Web UI, gRPC, and REST APIs.
+
+When you configure PKI authentication, by default, {product-title} uses the same port for PKI, web UI, gRPC, other single sign-on (SSO) providers, and REST APIs. You can also configure a separate port for PKI authentication by using a YAML configuration file to configure and expose endpoints.
+
+include::modules/configure-pki-authentication-portal.adoc[leveloffset=+1]
+
+include::modules/configure-pki-authentication-roxctl.adoc[leveloffset=+1]
+
+include::modules/update-authentication-keys-and-certificates.adoc[leveloffset=+1]
+
+include::modules/log-in-using-client-certificate.adoc.adoc[leveloffset=+1]
```

Comment:
When you added this, it has two `.adoc.adoc` extensions, so it isn't added properly still.